R/mipLineHistorical.R

```diff
@@ -58,7 +58,7 @@ mipLineHistorical <- function(x,x_hist=NULL,color.dim="identifier",linetype.dim=
   if(all(is.na(x$scenario))) x$scenario <- ""
   if(all(is.na(x$model))) x$model <- ""
   if (! "identifier" %in% names(x)) x$identifier <- identifierModelScen(x)
-  color.dim.name <- paste(c(color.dim.name, attr(x$identifier, "deletedinfo")), collapse = " ")
+  if (is.null(color.dim.name)) color.dim.name <- c(attr(x$identifier, "deletedinfo"), "Model output")[[1]]
```

Comment:
Uh, is that an [Elvis operator](https://en.wikipedia.org/wiki/Elvis_operator) for R? I was always wondering how to do that!

Reply:
I never heard of "Elvis operator", but this does not seem to be such an implementation, because it does not return `attr(x$identifier, "deletedinfo")` if it evaluates to `TRUE`, but if it does not evaluate to `NULL`.

Reply:
So it seems this is a [Null coalescing operating](https://en.wikipedia.org/wiki/Null_coalescing_operator).

Reply:
`c(intersect(a, TRUE), b)[[1]]` is Elvis, I guess :)